docs/guide/repository.md

```diff
@@ -656,38 +539,26 @@ When using Memcache cache backend, you *may* use inMemory to reduce network traf
 
 ##### Example with Memcache(d)
 
-Note that `app/config/config.yml` contains the default stash configuration. To apply the configuration below, make sure you update the existing configuration, or remove it if you want to use another configuration file.
-
 ``` yaml
-stash:
-    caches:
-        default:
-            drivers: [ Memcache ]
-            inMemory: true
-            registerDoctrineAdapter: false
-            Memcache:
-                prefix_key: ezdemo_
-                retry_timeout: 1
-                servers:
-                    -
-                        server: 127.0.0.1
-                        port: 11211
+services:
+    cache.memcached:
+        parent: cache.adapter.memcached
+        tags:
+            - name: cache.pool
+              clearer: cache.app_clearer
+              provider: 'memcached://user:pass@localhost?weight=33'
 ```
 
 !!! caution "Connection errors issue"
 
-    If memcached does display connection errors when using the default (ascii) protocol, then switching to binary protocol *(in the stash configuration and memcached daemon)* should resolve the issue.
+    If memcached does display connection errors when using the default (ascii) protocol, then switching to binary protocol *(in the configuration and memcached daemon)* should resolve the issue.
 
 ## Usage
 
 ### Using Cache Service
 
 Using the internal cache service allows you to use an interface and to not have to care whether the system has been configured to place the cache in Memcached or on File system. And as eZ Platform requires that instances use a cluster-aware cache in Cluster setup, you can safely assume your cache is shared *(and invalidated)* across all web servers.
 
-!!! caution "Interface will change in the future"
-
-    Current implementation uses a caching library called [Stash](http://stash.tedivm.com/), via [StashBundle](https://github.com/tedivm/TedivmStashBundle). We plan to move to a PSR-6 compatible cache service capable of supporting cache Tagging and multi get/set in the future, when that happens the interface of the cache service will change!
-
```

Comment:
the same caution can be kept for usage of ``Symfony\Component\Cache\Adapter\TagAwareAdapter`, type hinting on `Psr\Cache\CacheItemPoolInterface` is however quite safe.